FE/src/components/header/gnb/menuBar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as Styled from 'components/header/gnb/menuBar/menubar.style';
+
+function MenuBar() {
+  type MenuType = {
+    id: number;
+    title: string;
+  };
+
+  const menuData: MenuType[] = [
+    { id: 1, title: '숙소' },
+    { id: 2, title: '체험' },
+    { id: 3, title: '온라인 체험' },
+  ];
```

Comment:
```suggestion
  type MenuType = {
    id: number;
    title: string;
  };

  const menuData: MenuType[] = [
    { id: 1, title: '숙소' },
    { id: 2, title: '체험' },
    { id: 3, title: '온라인 체험' },
  ];
```

2개는 컴포넌트 내부가 아니여도 상수로써 관리해도 괜찮을 것 같습니다!